pandas/tests/reshape/merge/test_merge.py

```diff
@@ -2197,6 +2195,39 @@ def test_merge_series(on, left_on, right_on, left_index, right_index, nm):
             )
 
 
+@pytest.mark.parametrize(
+    "on,left_on,right_on,left_index,right_index,name",
+    [
+        (["outer", "inner"], None, None, False, False, ("B", "C")),
+        (None, None, ["outer", "inner"], True, False, ("B", "C")),
```

Comment:
Can you remove the unnecessary parametrizations?

Reply:
Is it necessary to parametrize at all?

Reply:
I can't think of an example

Reply:
A regular test is sufficient then, just something that reproduced the bug before